clickhouse-http-client/src/test/java/com/clickhouse/client/http/ClickHouseHttpClientTest.java

```diff
@@ -193,7 +193,8 @@ public void testUserAgent() throws Exception {
              ClickHouseResponse response = newRequest(client, server)
                      .query("select http_user_agent from system.query_log where query='select ''" + uuid + "'''")
                      .executeAndWait()) {
-            Assert.assertEquals(response.firstRecord().getValue(0).asString(), "MyCustomClient");
+            String result = response.firstRecord().getValue(0).asString();
```

Comment:
The test was green before your changes, wasn't it?
Should we add tests for version, `rv`, `jvm`, etc.?